README.md

```diff
@@ -234,18 +239,11 @@ if err != nil {
 }
 ```
 
-### Generate Token and Token Request
-
-```go
-client.Auth.RequestToken()
-client.Auth.CreateTokenRequest()
```

Comment:
Are these no longer relevant?

Reply:
Those weren't really usage examples. They lack arguments and don't use the return values.

Also, they arguably aren't relevant enough for most Go users, who are likely to prefer key auth given Go runs (in the overwhelming majority of cases) on private servers. So they're beyond the scope of an introductory README.